packages/esm-service-queues-app/src/modals/call-modal/call-queue-entry.modal.tsx

```diff
@@ -22,6 +23,9 @@ const CallQueueEntryModal: React.FC<CallQueueEntryModalProps> = ({ closeModal, q
   const { t } = useTranslation();
   const config = useConfig<ConfigObject>();
   const defaultTransitionStatus = config.concepts.defaultTransitionStatus;
+  const session = useSession();
+  const currentProviderUuid = session?.currentProvider?.uuid;
+  const canWork = canProviderWorkOnQueueEntry(queueEntry, currentProviderUuid);
```

Comment:
This should only be called or enabled if an implementation is opting-in to queue entries assigned to providers.

Reply:
But also shouldn't whether or not a provider can call a specific patient affect whether the call button / link is displayed at all and not just display some kind of text and disable calling them in the modal?